pages/index.vue

```diff
@@ -385,6 +385,12 @@ export default {
     padding-left: 0 !important;
     text-align: center;
     width: 100%;
+
+    h1,
+    h3,
+    p {
+      padding: 0 5vw;
+    }
```

Comment:
See image for my review of line 315; this also makes everything look like it's going to the left on larger screens

Reply:
Oddly enough my comment for said line 315 didn't save, lol. 

It still looks strange on 4k screens, albeit not as strange.
![image](https://i.imgur.com/yrVIzWH.png)

Reply:
Everything should look better on 4k screens after aed1a2c